README.md

```diff
@@ -1,2 +1,6 @@
 # hello-world
 CS 633 Homework
+
```

Comment:
Left a comment!

Reply:
Thank you David for invite
This is just an exercise for those who never used Git